### ISSUE TITLE
[Bug]: ScrollView seems broken

### ISSUE THREAD

**Issue by bcardarella** [liveview-native/liveview-client-swiftui#1185]
### What happened?

I tried the following in LVN:

```
<ScrollView>
  <Text><%= @walloftext %></Text>
</ScrollView>
```

and nothing renders

if I do this directly in SwiftUI

```swift
ScrollView {
  Text("""
... wall of text
""")
}
```

It renders just fine

### Library Version

latest

### Xcode Version

latest

### Swift Version

latest

### On which device or simulator are you running into the problem?

iPhone

### Target Device Operating System Version

latest iOS

**Comment by carson-katri** [comment]
I'm not able to replicate this. Was there anything else in the template that could have impacted it?

**Comment by bcardarella** [comment]
Log:

```
Connecting to http://localhost:4000/top/
nw_socket_handle_socket_event [C1.1.1:2] Socket SO_ERROR 61
nw_socket_handle_socket_event [C2.1.1:2] Socket SO_ERROR 61
[Socket] Opened
[LiveReload] attempting to connect...
nw_socket_handle_socket_event [C3.1.1:2] Socket SO_ERROR 61
[LiveReload] connected to channel
[Socket] SwiftPhoenixClient: push, Sending ["6","6","lv:phx-F6Aztq788-sHe7gF","phx_join",{"session":"SFMyNTY.g2gDaAJhBXQAAAAIdwJpZG0AAAAUcGh4LUY2QXp0cTc4OC1zSGU3Z0Z3B3Nlc3Npb250AAAAAHcKcGFyZW50X3BpZHcDbmlsdwZyb3V0ZXJ3GEVsaXhpci5Mdm5EZW1vV2ViLlJvdXRlcncEdmlld3cZRWxpeGlyLkx2bkRlbW9XZWIuVG9wTGl2ZXcIcm9vdF9waWR3A25pbHcJcm9vdF92aWV3dxlFbGl4aXIuTHZuRGVtb1dlYi5Ub3BMaXZldwxsaXZlX3Nlc3Npb25oAncHZGVmYXVsdG4IANyW3MbkMqAXbgYAN9X9X4wBYgABUYA.z-lzezFqLkT1bNT_h-OfSbpWEl7US8-Q-Zpuik8-yyM","url":"http:\/\/localhost:4000\/top\/","params":{"_mounts":0,"_lvn":{"bundle_id":"com.myorg.LvnDemo","os":"iOS","os_version":"17.0.1","target":"phone","app_version":"1.0","app_build":"1.0","format":"swiftui"},"_csrf_token":"DUEtFVxiKFkgbgY0MDwqM3QkMidFWSQPZwYFeSfls85zfQgWMSdmhoKx"},"static":"SFMyNTY.g2gDaAJhBXQAAAADdwJpZG0AAAAUcGh4LUY2QXp0cTc4OC1zSGU3Z0Z3BWZsYXNodAAAAAB3CmFzc2lnbl9uZXdqbgYAN9X9X4wBYgABUYA.70N6vn5Qigty-0wHwKfJZIse64Pqu8tkJNc3mKk-8H4"}]
[Socket] SwiftPhoenixClient: receive , ["6","6","lv:phx-F6Aztq788-sHe7gF","phx_reply",{"status":"ok","response":{"rendered":{"0":" body=\"%{}\"","1":{"0":" body=\"%{&quot;code&quot; =&gt; [{:monospaced, [file: &quot;/Users/bcardarella/projects/lvn_demo/lib/lvn_demo_web/styles/top_styles.ex&quot;, line: 6, module: LvnDemo.TopStyles], [true]}]}\"","1":" verbatim=\"Processes: 881 total, 3 running, 878 sleeping, 5344 threads \n2023/12/12 16:45:20\nLoad Avg: 10.10, 11.48, 10.26 \nCPU usage: 26.53% user, 34.11% sys, 39.35% idle \nSharedLibs: 521M resident, 100M data, 34M linkedit.\nMemRegions: 934257 total, 7137M resident, 303M private, 2857M shared.\nPhysMem: 23G used (4038M wired, 6919M compressor), 107M unused.\nVM: 410T vsize, 4723M framework vsize, 1891512(0) swapins, 3140477(0) swapouts.\nNetworks: packets: 916839200/1011G in, 253599521/74G out.\nDisks: 269443296/5216G read, 1836923571/13T written.\n\nPID    COMMAND          %CPU TIME     #TH   #WQ #PORTS MEM   PURG  CMPRS PGRP  PPID  STATE    BOOSTS        %CPU_ME %CPU_OTHRS UID FAULTS    COW      MSGSENT    MSGRECV    SYSBSD     SYSMACH    CSW        PAGEINS  IDLEW     POWER INSTRS CYCLES USER                  #MREGS RPRVT VPRVT VSIZE KPRVT KSHRD\n99814  gamecontrollerag 0.0  01:31.97 3     2   48     3553K 0B    2448K 99814 1     sleeping *0[1]         0.00000 0.00000    501 177454    101      475923     1096       3576695    1904439    3283254    4        20366     0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n99813  gamecontrollerd  0.0  01:47.86 3     2   117    4225K 0B    1552K 99813 1     sleeping *3[64]        0.00000 0.00000    247 438678    130      1254001    311294     3863203    3046062    3275156    867      19596     0.0   0      0      _gamecontrollerd      N/A    N/A   N/A   N/A   N/A   N/A  \n99502  netbiosd         0.0  00:02.74 2     1   25     3105K 0B    2752K 99502 1     sleeping *0[1]         0.00000 0.00000    222 43466     101      4681       1876       109524     10077      74352      493      368       0.0   0      0      _netbios              N/A    N/A   N/A   N/A   N/A   N/A  \n99499  flyctl           0.0  24:04.12 14    1   54     32M   0B    14M   99499 1     sleeping *0[1]         0.00000 0.00000    501 1410585   462      14141      1653       57687078   29532      42702060   2744     502585    0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n99169  erl_child_setup  0.0  00:00.07 1     0   11     1041K 0B    864K  99169 98865 sleeping *0[1]         0.00000 0.00000    501 6427      1936     596        297        5824       355        831        20       9         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n99068  AXVisualSupportA 0.0  33:06.77 7     1   281    13M   0B    7392K 99068 1     sleeping *6[130]       0.00000 0.00000    501 893647    196      56569552   16795935   39899855   130247387  25113021   417      97312     0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98870  inet_gethost     0.0  00:00.00 1     0   12     961K  0B    768K  98868 98868 sleeping *0[1]         0.00000 0.00000    501 85        27       21         7          49         36         12         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98869  inet_gethost     0.0  00:00.00 1     0   12     1025K 0B    832K  98868 98868 sleeping *0[1]         0.00000 0.00000    501 90        30       22         7          134        38         23         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98868  inet_gethost     0.0  00:00.00 1     0   11     849K  0B    672K  98868 98828 sleeping *0[1]         0.00000 0.00000    501 221       67       30         15         182        70         22         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98867  bash             0.0  00:00.00 1     0   8      817K  0B    640K  98864 98866 sleeping *0[1]         0.00000 0.00000    501 99        23       10         5          22         16         2          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98866  bash             0.0  00:00.00 1     0   8      865K  0B    688K  98864 98864 sleeping *0[1]         0.00000 0.00000    501 114       43       12         6          17         19         4          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98865  beam.smp         0.0  04:00.06 32    0   54     82M   0B    80M   98864 98864 sleeping *0[1]         0.00000 0.00000    501 445672    917      391        166        7242165    49467806   7127268    113      212943    0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98864  bash             0.0  00:00.00 1     0   11     833K  0B    672K  98864 98828 sleeping *0[1]         0.00000 0.00000    501 353       91       46         23         348        111        17         5        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98828  erl_child_setup  0.0  00:00.00 1     0   11     785K  0B    624K  98828 98827 sleeping *0[1]         0.00000 0.00000    501 248       81       32         16         179        67         10         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98827  beam.smp         0.0  03:00.00 32    0   55     128M  0B    125M  92202 1     sleeping *0[1]         0.00000 0.00000    501 970641    150      99         30         6353219    12066833   5070432    315      206653    0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98617  erl_child_setup  0.0  00:00.00 1     0   11     913K  0B    736K  98617 98317 sleeping *0[1]         0.00000 0.00000    501 299       104      38         18         244        72         18         3        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98430  inet_gethost     0.0  00:00.00 1     0   12     993K  0B    800K  98316 98316 sleeping *0[1]         0.00000 0.00000    501 86        23       22         7          104        38         21         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98429  inet_gethost     0.0  00:00.00 1     0   12     1025K 0B    832K  98316 98316 sleeping *0[1]         0.00000 0.00000    501 90        30       22         7          80         38         19         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98319  bash             0.0  00:00.00 1     0   8      785K  0B    608K  98315 98318 sleeping *0[1]         0.00000 0.00000    501 97        23       10         5          22         15         1          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98318  bash             0.0  00:00.00 1     0   8      833K  0B    656K  98315 98315 sleeping *0[1]         0.00000 0.00000    501 112       43       12         6          17         18         1          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98317  beam.smp         0.0  02:50.97 32    0   55     100M  0B    97M   98315 98315 sleeping *0[1]         0.00000 0.00000    501 446598    916      385        166        5087278    6765900    5107187    59       209173    0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98316  inet_gethost     0.0  00:00.00 1     0   11     833K  0B    672K  98316 98311 sleeping *0[1]         0.00000 0.00000    501 222       66       32         15         182        70         16         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98315  bash             0.0  00:00.00 1     0   11     833K  0B    672K  98315 98311 sleeping *0[1]         0.00000 0.00000    501 353       91       47         23         348        111        12         7        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98311  erl_child_setup  0.0  00:00.00 1     0   11     785K  0B    624K  98311 98309 sleeping *0[1]         0.00000 0.00000    501 243       81       34         16         179        66         13         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n98309  beam.smp         0.0  02:34.85 32    0   54     92M   0B    89M   92202 1     sleeping *0[1]         0.00000 0.00000    501 226403    150      111        30         5664507    3116049    4600434    82       207200    0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n97977  Brave Browser He 0.0  00:05.92 18    1   152    43M   0B    38M   64959 64959 sleeping *0[5]         0.00000 0.00000    501 285260    824      48763      23566      43993      96851      77066      32       433       0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n93747  Authy Desktop He 0.0  00:10.38 18    1   201    156M  0B    144M  93740 93740 sleeping *0[3]         0.00000 0.00000    501 313162    524      83379      46800      88985      84750      49224      2904     65        0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n93746  Authy Desktop He 0.0  00:02.25 16    1   167    72M   0B    70M   93740 93740 sleeping *0[1]         0.00000 0.00000    501 57086     521      32598      16370      32424      22508      19859      1825     38        0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n93745  Authy Desktop He 0.0  00:00.42 8     1   90     20M   0B    19M   93740 93740 sleeping *0[2]         0.00000 0.00000    501 23062     480      10478      5396       6457       8515       4597       462      45        0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n93743  Authy Desktop He 0.0  00:01.19 9     1   183    82M   0B    39M   93740 93740 sleeping *1[11]        0.00000 0.00000    501 49119     537      16362      8417       23469      31383      13785      2852     92        0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n93740  Authy Desktop    0.0  00:17.64 28    2   459    114M  0B    101M  93740 1     sleeping *0[256]       0.00000 0.00000    501 279964    638      121287     52723      179613     423016     150634     20700    1050      0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n93072  erl_child_setup  0.0  00:00.00 1     0   11     849K  0B    672K  93072 92771 sleeping *0[1]         0.00000 0.00000    501 197       52       28         13         144        65         6          3        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92776  inet_gethost     0.0  00:00.00 1     0   12     961K  0B    752K  92774 92774 sleeping *0[1]         0.00000 0.00000    501 82        21       21         7          49         36         14         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92775  inet_gethost     0.0  00:00.00 1     0   12     993K  0B    784K  92774 92774 sleeping *0[1]         0.00000 0.00000    501 88        29       22         7          134        37         24         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92774  inet_gethost     0.0  00:00.00 1     0   11     865K  0B    672K  92774 92759 sleeping *0[1]         0.00000 0.00000    501 224       69       30         15         182        69         11         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92773  bash             0.0  00:00.00 1     0   8      849K  0B    656K  92770 92772 sleeping *0[1]         0.00000 0.00000    501 100       23       10         5          22         16         1          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92772  bash             0.0  00:00.00 1     0   8      897K  0B    704K  92770 92770 sleeping *0[1]         0.00000 0.00000    501 113       42       12         6          17         19         2          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92771  beam.smp         0.0  00:14.07 32    0   55     73M   0B    71M   92770 92770 sleeping *0[1]         0.00000 0.00000    501 182609    919      363        166        196569     2541476    425580     43       374       0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92770  bash             0.0  00:00.00 1     0   11     1009K 0B    816K  92770 92759 sleeping *0[1]         0.00000 0.00000    501 365       92       46         23         380        112        5          4        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92759  erl_child_setup  0.0  00:00.00 1     0   11     849K  0B    656K  92759 92758 sleeping *0[1]         0.00000 0.00000    501 251       83       32         16         179        67         10         1        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92758  beam.smp         0.0  00:04.85 32    0   54     97M   0B    94M   41580 92653 sleeping *0[1]         0.00000 0.00000    501 32032     150      76         30         347956     2067728    323910     12       381       0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92654  zsh              0.0  00:00.12 1     0   22     4865K 0B    4528K 92654 41595 sleeping *0[1]         0.00000 0.00000    501 4602      1360     231        110        27235      317        846        68       0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92653  VSCodium Helper  0.0  67:14.39 17    1   76     82M   0B    63M   41580 41580 sleeping *0[1]         0.00000 0.00000    501 68908     548      1107143    9951       1291578910 4434427    80179211   128      11184     0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92652  VSCodium Helper  0.0  00:01.51 20    1   97     21M   0B    18M   41580 41580 sleeping *0[1]         0.00000 0.00000    501 19304     462      10900      293        60491      54022      41689      11       164       0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n92651  VSCodium Helper  0.0  00:58.84 18    1   205    140M  0B    88M   41580 41580 sleeping *0[5]         0.00000 0.00000    501 467738    491      756691     378492     785512     1471239    974786     411      65        0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n90275  cloudd           0.0  06:12.45 5     2   162    17M   0B    5984K 90275 1     sleeping *762[49467]   0.00000 0.00000    501 1217653   238      1388031    603940     8444224    3249408    3711830    4509     8781      0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n89526  zsh              0.0  00:01.44 1     0   22     7105K 0B    6736K 89526 89525 sleeping *0[1]         0.00000 0.00000    501 37693     6296     910        447        176559     659        5267       296      0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n89525  login            0.0  00:00.01 2     1   32     3025K 0B    2640K 89525 4987  sleeping *0[9]         0.00000 0.00000    0   855       109      77         31         2436       223        128        14       0         0.0   0      0      root                  N/A    N/A   N/A   N/A   N/A   N/A  \n85006  epmd             0.0  00:43.11 1     0   8      3777K 0B    3520K 85005 1     sleeping *0[1]         0.00000 0.00000    501 3833      23       15         5          955962     21         784739     4        51506     0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n84966  tccd             0.0  00:34.10 2     1   64     13M   0B    11M   84966 1     sleeping *0[12805]     0.00000 0.00000    501 149861    127      85640      91626      2009620    248539     254435     49       18        0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n82134  erl_child_setup  0.0  00:00.00 1     0   11     881K  0B    704K  82134 81852 sleeping *0[1]         0.00000 0.00000    501 277       93       36         17         224        70         19         4        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n81857  inet_gethost     0.0  00:00.00 1     0   12     1009K 0B    800K  81853 81853 sleeping *0[1]         0.00000 0.00000    501 91        30       22         7          104        36         19         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n81856  inet_gethost     0.0  00:00.00 1     0   12     1089K 0B    880K  81853 81853 sleeping *0[1]         0.00000 0.00000    501 96        32       22         7          80         39         20         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n81855  bash             0.0  00:00.00 1     0   8      849K  0B    656K  81851 81854 sleeping *0[1]         0.00000 0.00000    501 101       29       10         5          22         15         1          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n81854  bash             0.0  00:00.00 1     0   8      913K  0B    720K  81851 81851 sleeping *0[1]         0.00000 0.00000    501 116       45       12         6          17         18         5          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n81853  inet_gethost     0.0  00:00.00 1     0   11     881K  0B    704K  81853 81840 sleeping *0[1]         0.00000 0.00000    501 227       69       30         15         182        69         24         4        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n81852  beam.smp         0.0  00:04.62 32    0   55     70M   0B    69M   81851 81851 sleeping *0[1]         0.00000 0.00000    501 28907     922      364        166        184795     2048307    288325     302      511       0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n81851  bash             0.0  00:00.00 1     0   11     1137K 0B    928K  81851 81840 sleeping *0[1]         0.00000 0.00000    501 371       90       46         23         380        116        12         13       0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n81840  erl_child_setup  0.0  00:00.00 1     0   11     849K  0B    656K  81840 81836 sleeping *0[1]         0.00000 0.00000    501 254       85       32         16         179        67         12         3        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n81836  beam.smp         0.0  00:03.09 32    0   54     68M   0B    67M   41580 68084 sleeping *0[1]         0.00000 0.00000    501 27124     150      79         30         180738     1049790    266951     178      527       0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n80969  erl_child_setup  0.0  00:00.00 1     0   11     897K  0B    704K  80969 80666 sleeping *0[1]         0.00000 0.00000    501 423       134      44         21         304        75         36         1        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n80668  bash             0.0  00:00.00 1     0   8      785K  0B    608K  80665 80667 sleeping *0[1]         0.00000 0.00000    501 95        21       10         5          22         15         1          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n80667  bash             0.0  00:00.00 1     0   8      849K  0B    656K  80665 80665 sleeping *0[1]         0.00000 0.00000    501 109       40       12         6          17         17         2          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n80666  beam.smp         0.0  01:05.28 32    0   55     125M  0B    123M  80665 80665 sleeping *0[1]         0.00000 0.00000    501 525776    931      366        166        1140740    6446647    2811184    37       7924      0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n80665  bash             0.0  00:00.00 1     0   11     1057K 0B    848K  80665 52353 sleeping *0[1]         0.00000 0.00000    501 365       89       46         23         380        115        6          4        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n80392  Brave Browser He 0.0  38:31.16 21    1   305    245M  0B    81M   64959 64959 sleeping *0[8]         0.00000 0.00000    501 17197861  4411     3859505    1780403    18296358   10842323   13570020   55       68036     0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n79668  USBAgent         0.0  00:00.08 2     1   30     2817K 0B    2464K 79668 1     sleeping  0[0]         0.00000 0.00000    501 1829      96       160        84         638        382        417        12       1         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n78656  cfprefsd         0.0  05:39.09 3     2   495    4993K 0B    1728K 78656 1     sleeping *0[459931]    0.00000 0.00000    501 362867    85       1045614    1075814    12448535   2494383    2422630    3        1690      0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n78654  cfprefsd         0.0  04:17.16 3     2   652    4081K 0B    1408K 78654 1     sleeping *0[543216]    0.00000 0.00000    0   483190    83       435149     503769     9309780    1502457    1837567    3        2327      0.0   0      0      root                  N/A    N/A   N/A   N/A   N/A   N/A  \n78195  UserNotification 0.0  00:13.66 3     1   206    16M   0B    12M   78195 1     sleeping *0[1]         0.00000 0.00000    501 682325    191      149830     66219      190259     532069     272059     182      171       0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n76452  SimAudioProcesso 0.0  00:01.39 5     1   302    9026K 0B    5712K 76452 1     sleeping *0[1622]      0.00000 0.00000    501 25165     179      37555      29606      163095     122195     74666      50       3         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n76450  SimStreamProcess 0.0  00:00.27 3     1   188    11M   0B    9152K 76450 1     sleeping *0[309]       0.00000 0.00000    501 19562     195      4778       2579       8970       10944      6109       72       1         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n76204  erl_child_setup  0.0  00:00.00 1     0   11     897K  0B    720K  76204 75606 sleeping *0[1]         0.00000 0.00000    501 273       89       36         17         224        72         24         3        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n75610  bash             0.0  00:00.00 1     0   8      881K  0B    704K  75604 75608 sleeping *0[1]         0.00000 0.00000    501 101       25       10         5          22         19         16         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n75608  bash             0.0  00:00.00 1     0   8      833K  0B    656K  75604 75604 sleeping *0[1]         0.00000 0.00000    501 110       43       12         6          17         17         4          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n75606  beam.smp         0.0  01:27.59 32    0   55     68M   0B    66M   75604 75604 sleeping *0[1]         0.00000 0.00000    501 187987    915      374        166        2791660    3876609    2625217    43       110846    0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n75604  bash             0.0  00:00.00 1     0   11     913K  0B    736K  75604 72751 sleeping *0[1]         0.00000 0.00000    501 358       89       46         23         348        113        38         6        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n75157  ScreenTimeAgent  0.0  05:36.72 2     1   341    17M   0B    17M   75157 1     sleeping *0[11740]     0.00000 0.00000    501 1262439   234      124967     121148     11053466   11706069   1311708    3863     504       0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n74793  appstoreagent    0.0  13:51.18 5     2   521    27M   0B    24M   74793 1     sleeping *0[2754]      0.00000 0.00000    501 2556298   370      1236521    645953     8652499    2033001    4874043    11527    29068     0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n73359  Brave Browser He 0.0  00:23.35 18    1   227    63M   0B    52M   64959 64959 sleeping *0[7]         0.00000 0.00000    501 796311    1069     180730     96329      224621     350299     286985     0        2459      0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n73356  Brave Browser He 0.0  00:18.43 18    1   142    40M   0B    27M   64959 64959 sleeping *0[4]         0.00000 0.00000    501 464707    1069     258551     143455     172062     368214     248229     3        2269      0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n72945  inet_gethost     0.0  00:00.00 1     0   12     929K  0B    736K  72935 72935 sleeping *0[1]         0.00000 0.00000    501 81        23       21         7          49         35         10         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n72942  inet_gethost     0.0  00:00.13 1     0   12     1233K 0B    1024K 72935 72935 sleeping *0[1]         0.00000 0.00000    501 110       28       22         7          18294      44         2813       0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n72935  inet_gethost     0.0  00:00.04 1     0   11     865K  0B    688K  72935 72751 sleeping *0[1]         0.00000 0.00000    501 223       69       30         15         7580       68         1707       0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n72751  erl_child_setup  0.0  00:00.00 1     0   11     817K  0B    640K  72751 72739 sleeping *0[1]         0.00000 0.00000    501 332       111      38         19         236        70         26         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n72739  beam.smp         0.0  01:31.10 32    0   55     110M  0B    108M  72623 1     sleeping *0[1]         0.00000 0.00000    501 143215    163      83         30         3040168    5089721    2694716    143      111791    0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n71212  Brave Browser He 0.0  00:12.90 18    1   199    86M   0B    81M   64959 64959 sleeping *0[6]         0.00000 0.00000    501 310432    2994     70731      31616      114522     146544     135810     19       497       0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69829  erl_child_setup  0.0  00:00.00 1     0   11     865K  0B    688K  69829 69529 sleeping *0[1]         0.00000 0.00000    501 200       53       28         13         144        65         7          3        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69822  installd         0.0  01:41:43 2     1   811    30M   0B    30M   69822 1     sleeping *0[749]       0.00000 0.00000    0   991600    225      554011     283556     74461100   463764     17977500   3981     180       0.0   0      0      root                  N/A    N/A   N/A   N/A   N/A   N/A  \n69821  system_installd  0.0  00:26.02 2     1   681    22M   0B    21M   69821 1     sleeping *0[390]       0.00000 0.00000    0   142983    168      308921     162218     862987     239998     127423     275      147       0.0   0      0      root                  N/A    N/A   N/A   N/A   N/A   N/A  \n69533  inet_gethost     0.0  00:00.00 1     0   12     945K  0B    752K  69528 69528 sleeping *0[1]         0.00000 0.00000    501 86        27       21         7          49         34         11         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69532  inet_gethost     0.0  00:00.00 1     0   12     1025K 0B    832K  69528 69528 sleeping *0[1]         0.00000 0.00000    501 92        31       22         7          190        37         32         0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69531  bash             0.0  00:00.00 1     0   8      849K  0B    656K  69527 69530 sleeping *0[1]         0.00000 0.00000    501 100       23       10         5          22         16         1          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69530  bash             0.0  00:00.00 1     0   8      897K  0B    704K  69527 69527 sleeping *0[1]         0.00000 0.00000    501 114       44       12         6          17         18         2          0        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69529  beam.smp         0.0  02:17.58 32    0   54     63M   0B    61M   69527 69527 sleeping *0[1]         0.00000 0.00000    501 228954    913      378        166        4629916    1170612    4298039    52       197012    0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69528  inet_gethost     0.0  00:00.00 1     0   11     897K  0B    720K  69528 69518 sleeping *0[1]         0.00000 0.00000    501 229       67       30         15         200        71         26         4        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69527  bash             0.0  00:00.00 1     0   11     1041K 0B    864K  69527 69518 sleeping *0[1]         0.00000 0.00000    501 372       90       46         23         380        113        39         5        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69518  erl_child_setup  0.0  00:00.00 1     0   11     801K  0B    640K  69518 69514 sleeping *0[1]         0.00000 0.00000    501 249       82       32         16         179        66         12         3        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n69514  beam.smp         0.0  02:31.05 32    0   54     100M  0B    98M   92202 1     sleeping *0[1]         0.00000 0.00000    501 236747    147      99         30         5170346    2409581    4771237    172      216712    0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n68444  top              0.0  00:00.86 1/1   0   16     6897K 0B    0B    68444 63889 running  *0[1]         0.00000 0.00000    0   1517      63       1920034    960016     3681       968078     295        1        0         0.0   0      0      root                  N/A    N/A   N/A   N/A   N/A   N/A  \n68440  parsec-fbf       0.0  00:00.10 5     4   45     10M   0B    0B    68440 65981 sleeping *0[2]         0.00000 0.00000    501 4987      405      179        44         1658       457        538        58       0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n68438  debugserver      0.0  00:00.11 8     2   55     7906K 0B    0B    68438 66235 sleeping *0[1]         0.00000 0.00000    501 24892     44       52303      26126      1773       50068      899        6        0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n68437  iOS              0.0  00:00.81 11    8   108    27M   96K   0B    68437 68438 sleeping *0[6]         0.00000 0.00000    501 16990     1427     2013       265        8246       4496       4275       138      0         0.0   0      0      bcardarella           N/A    N/A   N/A   N/A   N/A   N/A  \n68420  mdworker_shared  0.0  00:00.05 3     1   51     2881K 0B    2000K 68420 1     sleeping *0[1]         0.00000 0.00000    501 1572      123      384        160        607        460        168        11       0         0.0   0
```

**Comment by bcardarella** [comment]
Closing as @carson-katri and I discovered this to be a data truncation issue. It appears that somewhere between 150k and 200k of websocket frame size the message is being truncated and this causes the app to not render with the malformed document.

When we get LVN Core done with Channels this should hopefully go away.

As a matter of reference, the web's websocket has a limit of 9.22 exabytes https://www.rfc-editor.org/rfc/rfc6455#section-5.2
